<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Iowa</class>
 <widget class="QWidget" name="Iowa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>680</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="welcomePage">
    <widget class="QLabel" name="welcomeTitle">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>580</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Welcome to Iowa Gambling Task</string>
     </property>
    </widget>
    <widget class="QLabel" name="welcomeGif">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>480</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string>welcomeGif show here</string>
     </property>
    </widget>
    <widget class="QPushButton" name="welcomeNext">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>540</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="consentPage">
    <widget class="QLabel" name="consentLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>1000</width>
       <height>400</height>
      </rect>
     </property>
     <property name="text">
      <string>consent show here</string>
     </property>
    </widget>
    <widget class="QPushButton" name="consentNext">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>540</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="consentWarn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>510</y>
       <width>551</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>warnning</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>480</y>
       <width>551</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="consentAgree">
        <property name="text">
         <string>Yes, I agree</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="consentDisagree">
        <property name="text">
         <string>No, I disagree</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="demographicsPage">
    <widget class="QLabel" name="demoTitle">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>501</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Please fill in the basic information</string>
     </property>
    </widget>
    <widget class="QPushButton" name="demoNext">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>540</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>200</y>
       <width>491</width>
       <height>243</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="demoAge">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Age：</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="demoAgeSpin"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="demoGender">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Gender：</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="demoGenderCombo">
          <item>
           <property name="text">
            <string>Please select</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Male</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Female</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Prefer not to</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="demoEducation">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Education Level：</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="demoEducationCombo">
          <item>
           <property name="text">
            <string>Please select</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Secondray Education</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Undergraduate Programs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Postgraduate Programs (Master Degree)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Postgraduate Programs (PhD)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="demoGenderWarn">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>310</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="demoEducationWarn">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>410</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="demoAgeWarn">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>210</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="introducePage">
    <widget class="QLabel" name="introduceGif">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>190</y>
       <width>400</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string>introduceGif show here</string>
     </property>
    </widget>
    <widget class="QPushButton" name="introduceNext">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>540</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="introduceContent">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>601</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Test Introduction</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gamePage">
    <widget class="QPushButton" name="gameContinue">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>540</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
    <widget class="QLabel" name="deckA_">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>156</width>
       <height>209</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>unselected.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="deckB_">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>210</y>
       <width>156</width>
       <height>209</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>unselected.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="deckC_">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>210</y>
       <width>156</width>
       <height>209</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>unselected.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="deckD_">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>210</y>
       <width>156</width>
       <height>209</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>unselected.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="currentMoneyLabel">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current total：</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentMoney">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>160</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>money</string>
     </property>
    </widget>
    <widget class="QLabel" name="selectTip">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>490</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Please select a card</string>
     </property>
    </widget>
    <widget class="QLabel" name="gameGif">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>261</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>420</y>
       <width>151</width>
       <height>48</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="deckBResult">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>         DeckB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deckBResult_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>420</y>
       <width>151</width>
       <height>48</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="deckCResult">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>         DeckC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deckCResult_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>420</y>
       <width>151</width>
       <height>48</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="deckDResult">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>         DeckD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deckDResult_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>420</y>
       <width>151</width>
       <height>48</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="deckAResult">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>        DeckA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deckAResult_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>700</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>axis.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>82</y>
       <width>40</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cash</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>70</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Borrowed</string>
     </property>
    </widget>
    <widget class="QLabel" name="gameCashBar">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>82</y>
       <width>100</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Cash bar</string>
     </property>
    </widget>
    <widget class="QLabel" name="gameBorrowedBar">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>100</y>
       <width>100</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Borrowed bar</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentMoneyLabel_2">
     <property name="geometry">
      <rect>
       <x>483</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Round：</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentRound">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="finalMoneyPage">
    <widget class="QLabel" name="finalMonetTitle">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Your final money</string>
     </property>
    </widget>
    <widget class="QLabel" name="finalMonetGif">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>260</y>
       <width>351</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string>fainaMoneyGif show here</string>
     </property>
    </widget>
    <widget class="QPushButton" name="finalMonetNext">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>540</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="finalMoneyValue">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>190</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>final Money</string>
     </property>
    </widget>
    <widget class="QLabel" name="finalBar">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>731</width>
       <height>10</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="debriefPage">
    <widget class="QLabel" name="debriefTitle">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>90</y>
       <width>341</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>About this experiment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="debriefQuit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>540</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Quit</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="debriefContent">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>190</y>
       <width>621</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>549</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
